.github/ISSUE_TEMPLATE/feature-request.yaml

```
@@ -0,0 +1,67 @@
+name: Feature Request
+description: File a feature request or task
+title: "[FTR]: "
```

We already have the labels, I'd skip the prefix:

```suggestion
title: ""
```